.github/workflows/build-test.yml

```diff
@@ -80,9 +80,30 @@ jobs:
       - name: Fetch tags
         run: |
           git fetch --tags
-
+          
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v4
+        with:
+          php-version: '8.0'
```

Comment:
Install PHP stage is already there, You dont need to add additional Setup PHP.

This test uses a concept of `matrix` where the multiple version of PHP are used. And the current implementation of Installing PHP follows that.